resources/definitions/arjun_duplication.def.json

```diff
@@ -0,0 +1,49 @@
+{
+    "version": 2,
+    "name": "Arjun 300 Duplication",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Venkat Kamesh",
+        "manufacturer": "Sri Vignan Technologies",
+        "weight": 3,
+        "file_formats": "text/x-gcode",
+        "has_material": true,
+        "has_variants": true,
+        "preferred_variant_name": "0.4 mm",
```

Comment:
Found some mistakes when testing. This preferred variant name doesn't exist for this printer, so it chooses an arbitrary one. I think you meant to use `0.4 mm Nozzle` so I'll make a correction there.